docs/release-notes/2.38.0.md

```diff
@@ -0,0 +1,209 @@
+
+<!---
+!!! caution alert alert-warning "Important/Caution"
+    Crucial points that need emphasis:
+
+    - Important: A significant point that deserves emphasis.
+    - Caution: Used to mean 'Continue with care'.
+ --->
+
+
+## Release Highlights
+
+### Important security updates
+
+**RHEL 9**
+
+As security is paramount to us, starting with PMM 2.38.0, we have laid a solid and modernized enterprise-grade foundation for PMM. We have migrated our base operating system on which we build our base image from Enterprise Linux 7 ( CentOS 7) to Oracle Linux 9. Thus starting with PMM 2.38.0, we have made PMM publicly available on Oracle Linux 9.
+
+There are many benefits of this transition. The migration fixes a large number of CVEs and will see improved upstream response time to security-related issues. It also uses more modern libraries for our team to build on long with better compatibility with new technologies. 
+
+We will be rolling this out in phases. 
+
+The following users will be able to take advantage of this immediately:
+
+*New installations of PMM* 
+
+- New installations will pull down the Oracle Linux 9 container by default
+- Existing users that upgrade PMM using a full container replacement method will also pull the Oracle Linux 9 
+- Fresh AMI installations
+- Fresh OVF installations
+
+Users who will need to wait a bit longer:
+
+- Users that exclusively upgrade through the UI’s **Upgrade Now** button including:
+-- Docker users that use UI upgrade
+-- AMI users
+-- OVF users
+
+
+**Grafana**
+
+Grafana has been upgraded to 9.2.18 due to [CVE-2023-28119](https://github.com/advisories/GHSA-5mqj-xc49-246p) and [CVE-2023-1387](https://github.com/advisories/GHSA-c3h9-vpfv-3x4m).
+
+**Exporters** 
+
+Updated the mysqld_exporter, node_exporter, postgres_exporter, mongo_exporter to fix the CVE's.
+
+**ClickHouse plugin**
+
+We have migrated from the [Altinity Clickhouse datasource](https://grafana.com/grafana/plugins/vertamedia-clickhouse-datasource/) to the [Grafana Clickhouse datasource](https://grafana.com/grafana/plugins/grafana-clickhouse-datasource/) due to the vulnerabilities discovered. 
+
+Due to this migration, we have also updated our experimental Environment Overview Dashboard.
+
+!!! caution alert alert-warning "Important"
+    Though we have removed the datasource from PMM, we are still reusing the same name (ClickHouse) to keep the changes minimal. Thus, if you are using the old datasource in your dashboards, ensure that you upgrade the underlying queries that will pull the data from the new datasource.
+   
+Example
+
+The following example shows the query after migration:
+
+    ```sh
+    // Grafana ClickHouse datasource
+    SELECT period_start as t,
+    sum(m_query_time_sum)/sum(m_query_time_cnt) as QLatency,
+    environment
+    FROM pmm."metrics" 
+    WHERE $__timeFilter(period_start) AND environment != ''
+    GROUP BY t, environment
+    ORDER BY t    
+    ```
```

Comment:
Please replace this with more informative information provided at https://jira.percona.com/browse/PMM-12165, as it highlights the differences in queries.

Reply:
ok